server.js

```diff
@@ -322,14 +322,14 @@ export async function startServer() {
     );
   }
 
-  const httpServer = app.listen(3000, () => {
+  const httpServer = app.listen(PORT, () => {
     console.log(
-      `Server started at ${http2 ? "https" : "http"}://localhost:3000`,
+      `Server started at ${http2 ? "https" : "http"}://localhost:${PORT}`,
     );
   });
 
-  const playServer = play.listen(3001, () => {
-    console.log(`Play server started at http://localhost:3001`);
+  const playServer = play.listen(PLAY_PORT, () => {
+    console.log(`Playground server started on port ${PLAY_PORT}`);
```

Comment:
Nit: Could make clearer that this is not the Playground front end?

```suggestion
    console.log(`Playground back end started on port ${PLAY_PORT}`);
```